acct-group/gamemode/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>hurikhan77+bgo@gmail.com</email>
```

Comment:
(Is this still your e-mail address you use in bugzilla?)

Reply:
Yes... It's the only bugzilla account I have. But I use different addresses for "Signed-off-by" and authorship in Git. Is that a problem?

Reply:
It's not a problem, for some reason that e-mail ringed a bell and I wanted to make sure this wasn't the reason.

There were some maintainers who had switched their bugzilla e-mails while leaving old ones to metadata.xml -> bugs can't be assigned.

Reply:
But your bell worked correctly, we already discussed this topic. ;-)